app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :post
+
+  def update_likes_counter
```

Comment:
- [ ] Also both update..counter methods should be called before or after creating. By just creating the model wont help you to update the table. More information: [before_save...](https://guides.rubyonrails.org/active_record_callbacks.html)